ktor-server/ktor-server-plugins/ktor-server-i18n/jvm/src/io/ktor/i18n/Translate.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.i18n
+
+import io.ktor.server.routing.*
+import io.ktor.utils.io.charsets.*
+import java.util.*
+
+/**
+ * Translate a message key to an accepted language specified in HTTP request
+ */
+public fun RoutingContext.i18n(key: String): String {
+    val acceptedLanguages = call.attributes[ACCEPTED_LANGUAGES]
+    val availableLanguages = call.attributes[AVAILABLE_LANGUAGES]
+    val defaultLanguage = call.attributes[DEFAULT_LANGUAGE]
+
+    val bestMatchLanguage = acceptedLanguages.firstOrNull {
+        availableLanguages.contains(it.value)
+    }?.value ?: defaultLanguage
```

Comment:
I think we ought to be setting the `Content-Language` header somewhere in the response.  It'd make it easier for debugging and provide some browser support.

Reply:
Thanks, will fix